test/Rings/runtests.jl

```diff
@@ -1,3 +1,6 @@
+using Oscar
+using Test
```

Comment:
If we are `using Test` in basically every testfile, we do not check that it is already loaded in
https://github.com/oscar-system/Oscar.jl/blob/81b983d1df7ada2aca3641bae806f01722219799/src/utils/tests.jl#L23

Reply:
But we currently don't do it in ever testfile, just in the `runtests.jl` files...